assets/js/scripts.js

```diff
@@ -3503,231 +3503,200 @@ $.magnificPopup.registerModule(RETINA_NS, {
-                    return p.is('figure') ? p.find('figcaption').text() : item.el.find('img').attr('alt');
-                }
-            },
-            callbacks: {
-                beforeOpen: function () {
-                    this.st.image.markup = this.st.image.markup.replace('mfp-figure', 'mfp-figure mfp-with-anim');
-                    this.st.mainClass = 'mfp-zoom-in';
-                },
-                change: function () {
-                    var t = this.content.find('.mfp-title');
-                    if (t) {
-                        if (t.text() == "") {
-                            t.addClass('nobg');
-                        } else {
-                            t.removeClass('nobg');
-                        }
-                    }
-                }
-            }
-        });
-    });
+        init: function () {
+            FoundationOverrides.initFoundation();
+            FoundationOverrides.binds();
+        },
 
+        initFoundation: function () {
 
-    /* EVENTS */
+            // Initialize Foundation
+            Foundation.Orbit.defaults.navButtons = false;
 
-    var updateCurrentEventFilter = function (cal) {
-        var currentFilter = [];
-        $("#event-filter-dropdown input:checkbox:checked").each(function () {
-            currentFilter.push($(this).data('slug'));
-        });
-        cal.data('currentFilter', currentFilter);
-    };
+            $(document).foundation();
+
+        },
 
-    if ($.fullCalendar) {
+        binds: function () {
 
-        // Load large calendar
-        var cal = $('#calendar').fullCalendar({
-            firstDay: 1,
-            height: "auto",
+            // Don't follow links with data-open attribute
+            $("a[data-open]").on('click tap', function (e) {
+                e.preventDefault();
+            });
 
-            defaultView: Foundation.MediaQuery.atLeast('large') ? 'month' : 'listMonth',
+            // Add class to body when menu open, for styling purposes
+            $('[data-responsive-toggle]').on('toggled.zf.responsiveToggle', function () {
+                var body = $("body"), button = $('.wisv-menu-icon');
 
-            customButtons: {
-                filterButton: {
-                    text: "Filter",
+                if (body.hasClass("menu-open")) {
+                    body.removeClass("menu-open");
+                    button.removeClass('is-active');
+                } else {
+                    body.addClass("menu-open");
+                    button.addClass('is-active');
                 }
-            },
 
-            header: {
-                left: 'title',
-                center: '',
-                right: 'filterButton today prev, next'
-            },
+            });
+        }
 
-            events: {
-                url: '/wp-json/wp/v2/events/fullcalendar'
-            },
+    }
 
-            timeFormat: 'H:mm',
-            displayEventEnd: true,
+})(jQuery);
+
+var Gallery;
```

Comment:
Why is this also here and in a separate file? Same as Portal and FoundationOverrides?

Reply:
This is the concatened source file of all javascript we use, which is then minified into scripts.min.js and used in production. CHoice and calendar are not included, because they're only loaded when needed separately.